app/controllers/apipie/apipies_controller.rb

```diff
@@ -94,7 +94,7 @@ def get_language
 
     def authorized_doc
 
-      return @doc unless Apipie.configuration.authorize
```

Comment:
What about doing this check in separate line like this:

```ruby
return unless @doc
```

It would read a bit easier for me than having both conditions at once.